tests/test_diaPipe.py

```diff
@@ -292,6 +293,18 @@ def check_diaObjects(bbox, wcs, diaObjects):
         # Verify that no objects inside the bounding box were removed
         self.assertEqual(nIn1, nIn0)
 
+    def test_make_empty_catalog(self):
+        """Check that an empty catalog has the correct format.
+        """
+        tableName = "DiaObject"
```

Comment:
Does it also work with other tables, e.g. `DiaForcedSource`?

Reply:
Yes, it should work with any of the tables. I'll add a `for` loop over a few tables.